src/Domain/Customer/Model/CustomerAggregate/Customer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Domain.Customer.Model.CustomerAggregate.Validators;
+using Domain.SeedWork;
+
+namespace Domain.Customer.Model.CustomerAggregate;
+
+public sealed class Customer : Entity, IAggregatedRoot
+{
+    public Cpf Cpf { get; private set; }
+    public string Name { get; private set; }
+    public Email Email { get; private set; }
+
+    private Customer(Guid id, Cpf cpf, string name, Email email)
+    {
+        Id = id;
+        Cpf = cpf;
+        Name = name;
+        Email = email;
+    }
+
+    public static Customer New(string cpf, string name, string email)
+    {
+        var customer = new Customer(Guid.NewGuid(), cpf, name, email);
+
+        if (Validator.IsValid(customer, out var error) is false)
+            throw new DomainException(error);
+
+        return customer;
+    }
+
+    private static readonly IValidator<Customer> Validator = new CustomerValidator();
```

Comment:
tb n usuario propriedade estatica

Reply:
A ideia aqui, @mpatricio007, seria alocar um único _Validator_ na _Heap_, e melhorar o uso de memória.
Segundo a [documentação do C#](https://learn.microsoft.com/pt-br/dotnet/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members#static-members):

> "Existe apenas uma cópia de um membro estático, independentemente de quantas instâncias da classe forem criadas."